libr/core/cmd_anal.c

```diff
@@ -6729,6 +6729,11 @@ static int cmd_anal_all(RCore *core, const char *input) {
 			r_cons_break_timeout (r_config_get_i (core->config, "anal.timeout"));
 			r_core_anal_all (core);
 			rowlog_done (core);
+			// Run pending analysis immediately after analysis
+			// Usefull when running commands with ";" or via r2 -c,-i
+			if (core->anal && core->anal->cmdtail) {
+				r_core_cmd0 (core, "");
```

Comment:
looks like a hack :? what is this suposed to do? maybe good to just put the line that is needed to make this work instead of running no-command